draft-js-inline-toolbar-plugin/src/components/Toolbar/index.js

```diff
@@ -60,9 +61,19 @@ export default class Toolbar extends React.Component {
 
       if (!selectionRect) return;
 
+      const toolbarwidth = this.toolbar.clientWidth;
+
+      let left = (selectionRect.left - relativeRect.left) + (selectionRect.width / 2) - (toolbarwidth / 2);
+
+      if (left < padding) {
+        left = padding;
+      } else if (left > (document.documentElement.clientWidth - toolbarwidth + padding)) {
```

Comment:
This seems wrong. It takes into account the document width rather than the container width...